FSharp.Azure.StorageTypeProvider.sln

```diff
@@ -1,55 +0,0 @@
-Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
```

Comment:
Why did you remove the sln files?

Reply:
Sorry that was a mistake ... we be added again